consensus/ibft/ibft.go

```diff
@@ -448,10 +432,52 @@ func (i *Ibft) buildBlock(snap *Snapshot, parent *types.Header) (*types.Block, e
 	// is sealed after all the committed seals
 	block.Header.ComputeHash()
 
-	i.logger.Info("build block", "number", header.Number, "txns", len(txns))
+	i.logger.Info("build block", "number", header.Number, "txPoolLength", len(txns))
```

Comment:
What does `txPoolLength` means? len(txns) semms to be number of transactions in the block

Reply:
thats an unintended change, probably due to using rename refactoring in goland, will revert this

Reply:
[reverted](https://github.com/0xPolygon/polygon-sdk/pull/237/commits/ecdc987abb40a81960d8dbbf35b60fe59ef78c53)